// This file is part of mathtrader.
//
// Copyright (C) 2021 George Ioannidis
//
// mathtrader is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
//
// mathtrader is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// You should have received a copy of the GNU General Public License
// along with mathtrader. If not, see <http://www.gnu.org/licenses/>.

// Proto definition for an item.

syntax = "proto2";

package mathtrader;

// Message encapsulating an exchangeable item. Usually a board game, but can
// also be, e.g., a dummy item.
message Item {
  // The id assigned to the item by the OLWLG, e.g., "0007-PANDE".
  optional string id = 1;

  // Indicates whether the item is dummy or not.
  optional bool is_dummy = 2;

  extensions 100 to 199;
}
